src/Nethermind/Nethermind.JsonRpc/Modules/Eth/EthGasPriceConstants.cs

```diff
@@ -0,0 +1,15 @@
+using Nethermind.Int256;
+
+namespace Nethermind.JsonRpc.Modules.Eth
+{
+    public static class EthGasPriceConstants
+    {
+        public const int PercentileOfSortedTxs = 60; //Percentile of sortedTxList indexes to choose as gas price
+        public const int DefaultBlocksLimit = 20; //Limit for how many blocks we check txs in to add to sortedTxList
+        public const int DefaultGasPrice = 1; //Tx price added to sortedTxList for a block that has 0 txs (now adds 1 price)
+        public const int SoftTxLimit = 40; //If a block adds only 1 tx price, the block is not counted towards BlocksLimit if sortedTxList.Count() + blocksLeftToCheck <= SoftTxLimit (after price is added to sortedTxList) 
+        public const int DefaultBaseFee = 200; //Minimum price a transaction sender has to pay
```

Comment:
Why DefaultBaseFee = 200?
Why _maxGasPrice = 500?

Where did you find these consts?

Reply:
Default base Fee, I found in a Geth test (I removed it now), but maxGasPrice was part of the Geth code: https://github.com/ethereum/go-ethereum/blob/master/eth/gasprice/gasprice.go#L35.